numpy/core/tests/test_indexing.py

```diff
@@ -329,6 +329,21 @@ def test_trivial_fancy_out_of_bounds(self):
         assert_raises(IndexError, a.__getitem__, ind)
         assert_raises(IndexError, a.__setitem__, ind, 0)
 
+    def test_trivial_fancy_not_possible(self):
+        # Test that the fast path for trivial assignment is not incorrectly
+        # used when the index is not contiguous or 1D, see also gh-11467.
+        a = np.arange(6)
+        idx = np.arange(6).reshape(2, 1, 3)[:, :, 0].astype(np.intp)
```

Comment:
This voids the test, since `astype` makes the array contiguous.

You need to do `np.arange(6, dtype=np.intp)`

Reply:
Oops, yes :/, fixed.